src/Application/MappingRepository.php

```diff
@@ -4,12 +4,16 @@
 
 namespace ProfessionalWiki\WikibaseRDF\Application;
 
+use PermissionsError;
 use Wikibase\DataModel\Entity\EntityId;
 
 interface MappingRepository {
 
 	public function getMappings( EntityId $entityId ): MappingList;
 
+	/**
+	 * @throws PermissionsError
```

Comment:
IDE complained this exception was not thrown in `SaveMappingsUseCase`. Since this interface doesn't know about the `EntityContentRepository` implementation it seems like we need to declare it here.
![Screenshot_20220805_231346](https://user-images.githubusercontent.com/1428594/183195819-9e04cbfe-5b10-4ba8-9d0e-b377df411c2f.png)